EIPS/eip-6353.md

```diff
@@ -0,0 +1,333 @@
+---
+eip: 6353
+title: Charity token
+description: Extension of EIP-20 Token that can be partially donated to a charity project
+author: Aubay <blockchain-team@aubay.com>, BOCA Jeabby (@bjeabby1507), EL MERSHATI Laith (@lth-elm), KEMP Elia (@eliakemp)
+discussions-to: https://ethereum-magicians.org/t/erc20-charity-token/12617
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-05-13
+requires: 20
+---
+
+## Abstract
+
+An extension to the [EIP-20](./eip-20.md) standard token that allows Token owners to donate to some charity organization. The following standard allows for the implementation of the standard token to enforce charity donation by default. This standard is an extension of the [EIP-20](./eip-20.md) token to provide transfers to a third-party charity during transactions. This standard also allows token that support [EIP-20](./eip-20.md) interface, to have a standardized way of signalling charity information, registered by the token contract.
+
+## Motivation
+
+The initial idea is to allow Token owners to easily donate passively, in order to facilitate contributions to non-profit organizations and facilitate integration by giving a generalized implementation. Users can make an impact with their token and can contribute to achieving sustainable blockchain development. It is the possibility to donate simply with micro-donation during transactions. Projects can easily retrieve charity donations addresses and rate for a given [EIP-20](./eip-20.md) token, token holders can compare minimum rate donation offers allowed by token contract owners.
+
+## Specification
+
+**EIP-20 compliant contract MAY implement this ERC for automatizing Mirco payment during transactions.**
```

Comment:
```suggestion
```

You can't really force EIP-20 tokens to follow this standard, so the "MAY" is implied.